webrender_api/src/display_item.rs

```diff
@@ -113,19 +113,50 @@ pub struct ClipDisplayItem {
     pub image_mask: Option<ImageMask>,
 }
 
+/// Constraints for sticky frames for a single dimension.
+#[repr(C)]
 #[derive(Clone, Copy, Debug, Deserialize, PartialEq, Serialize)]
-pub struct StickyFrameDisplayItem {
-    pub id: ClipId,
-    pub sticky_frame_info: StickyFrameInfo,
+pub struct StickyOffsetConstraints {
+    /// The minimum offset for this frame, typically a negative value, which specifies how
+    /// far in the negative direction the sticky frame can offset its contents in this
+    /// dimension.
+    pub min_offset: f32,
```

Comment:
I think this would be slightly clearer if we called these fields `lower_bound` and `upper_bound` rather than `min_offset` and `max_offset`. The 'bound' naming also seems more consistent with the 'Constraints' naming on the struct. The comment could end up like this:
```
/// The lower bound for the frame's sticky offset. This is typically a negative value
/// which constrains how far in the negative direction the sticky frame can offset its
/// contents in this dimension.
```

Reply:
The only issue I have the with `bounds` terminology is that it isn't totally clear that this is dealing with offsets and not absolute or relative positions. Perhaps it makes sense to change the name of the data structure to something like StickyOffsetRange?

Reply:
I ended up going with `StickyOffsetBounds` here with `min` and `max`. I didn't want to use upper and lower, because their meanings are reversed visually (the lower bound specifies how far upward the sticky item can move). I wanted to avoid any possible confusion there.

Reply:
Ok, fair enough. I hadn't thought about the visual interpretation of upper and lower - you're right that it could get confusing :)